gsd/hoomd.py

```diff
@@ -60,7 +63,22 @@ class ConfigurationData(object):
     def __init__(self):
         self.step = None
         self.dimensions = None
-        self.box = None
+        self._box = None
+
+    @property
+    def box(self):
```

Comment:
Flake8 is throwing an error here. We either need a noqa or to move the box documentation here.